python/cudf/cudf/core/dataframe.py

```diff
@@ -3504,7 +3504,7 @@ def quantiles(self, q=0.5, interpolation="nearest"):
         if isinstance(q, numbers.Number):
             q_is_number = True
             q = [float(q)]
-        elif isinstance(q, (list, tuple)):
+        elif isinstance(q, (list, tuple, np.ndarray)):
```

Comment:
We could just use `pd.api.types.is_list_like()` here which encompasses these types and more.